### DISCUSSION TITLE
Static binaries in release

### DISCUSSION THREAD

**Discussion by loeffel-io** [d99kris/nchat#202] [General]
Would be amazing to have some static binaries in the releases

**Comment by d99kris** [comment]
Sorry, at this point packaging/building is outside the scope of the project.

**Comment by d99kris** [comment]
Pre-built binaries has been added for the latest release [v5.17.26](https://github.com/d99kris/nchat/releases/tag/v5.17.26), for installation steps see [Install Pre-built Binary](https://github.com/d99kris/nchat#install-pre-built-binary).

**Comment by loeffel-io** [comment]
🙌
